### ISSUE TITLE
What is this clusterfuck of a repo

### ISSUE THREAD

**Issue by AleanaA** [Pizzabelly/EasyRP#40]
With the build files actually not even working because of numberous typos in make files and the bash build script, the numerous, supposedly false, virus reports, and the whole repo not being maintained to any degree, why isn't anyone finding a better option?

I am 90% sure there's a better options than whatever this shit show is, and virus or not, it's probably not as easy as it says it is, especially if you're on linux or mac. 

[Hell, here's a random one I found googling Easy Rich Presence. Go use it instead.](https://github.com/justJS/SimplePresence)

**Comment by AleanaA** [comment]
To follow up on that, this is what's needed for rich presence in js.
``const client = require('discord-rich-presence')('app-id');
client.updatePresence({
  details: "required",
  state: "optional",
  argeImageKey: 'optional image',
  instance: true,
});``

All you need to do is install your dependencies, and run the script.
This can even be automated by a batch or bash script.

**Comment by AleanaA** [comment]
So to follow up on this, because this repo seems completely unmaintained and isn't cross platform with linux and mac, or at least not well, I made my own easy to setup rich presence which you can find [here](https://github.com/AleanaA/Easy-Rich-Presence)

**Comment by AleanaA** [comment]
:shrug: Isn't even really a project, but after the month this repo was up, none of these issues were touched, which led me, and probably others, to believe that this was entirely unmaintained. With that being the case, and seeing that the Linux build script just doesn't work, making an issue regarding my thoughts seemed appropriate, and my small repo for something similar to this was solely to have something that would work, regardless of your platform, and where people could get support for it.

It's nice to see you didn't just upload and abandon this completely like I've seen happen with other projects.

**Comment by Pizzabelly** [comment]
Yeah to be honest those build scripts we're terrible, I should just stick to cmake/meson lol. I understand your original thought of this being unmaintained (I would have thought the same) so no hard feelings :).

**Comment by GeorgeCY2** [comment]
and also there is a python edition to it tho

**Comment by edwodiculi** [comment]
hey i like this project :(
